src/langchain.ts

```diff
@@ -1,13 +1,13 @@
 import { BaseLLM } from 'langchain/llms/base';
 import { Client, Configuration } from './client.js';
 import { CallbackManagerForLLMRun } from 'langchain/callbacks';
-import { isNotEmptyArray, concatUnique } from './helpers/common.js';
+import { isNotEmptyArray, concatUnique, isNullish } from './helpers/common.js';
 import type { LLMResult, Generation } from 'langchain/schema';
 import type { GenerateOutput } from './client-types.js';
 import { GenerateInput } from './client-types.js';
 
 export interface GenAIModelOptions {
-  modelId: string;
+  modelId?: string | null;
```

Comment:
What's the reasoning behing allowing `null` here?

Reply:
Broader support for falsy values. In case the user will set the model and want it to be empty (dynamically for example) - they can use null.

Reply:
Broader support for falsy values. If the user sets the model and wants it to be empty (dynamically for example), they can use null.

Reply:
I see two interface inconsistencies:

1. other options has no such type support
2. `_modelType` returns `'default'` string (for technical reasons), not null

Both would be resolved if `default` literal type would be used instead of the `null`.

Reply:
I see, let's stick to the `undefined` for now and later add `null` support for optional options.